cli/flox/src/commands/environment.rs

```diff
@@ -1207,6 +1207,11 @@ impl Install {
     ) -> anyhow::Error {
         debug!("install error: {:?}", err);
 
+        subcommand_metric!(
+            "install",
+            "failed_packages" = packages.iter().map(|p| p.pkg_path.clone()).join(",")
```

Comment:
nit: this name is a bit misleading as some of these packages could successfully install